Makefile

```
@@ -286,6 +290,8 @@ lint-fix: ##@test Run code style checks and fix issues
 
 test: export TARGET := clojure
 test: ##@test Run tests once in NodeJS
+	nix/scripts/build.sh targets.status-go.library
```

Why do this if you can just make this target depend on the library target?
```suggestion
test: status-go-library ##@test Run tests once in NodeJS
	nix/scripts/build.sh targets.status-go.library
```